torch/csrc/jit/codegen/cuda/lower_utils.cpp

```diff
@@ -1,5 +1,6 @@
 #include <torch/csrc/jit/codegen/cuda/lower_utils.h>
 #include <torch/csrc/jit/codegen/cuda/ir_iostream.h>
+#include <torch/csrc/jit/codegen/cuda/lower_thread_predicate.h>
```

Comment:
nit: empty line between #include and namespace

Reply:
Thanks. Empty line added.